go.sum

```diff
@@ -3,9 +3,11 @@ github.com/golang/protobuf v1.4.0-rc.1.0.20200221234624-67d41d38c208/go.mod h1:x
 github.com/golang/protobuf v1.4.0-rc.2/go.mod h1:LlEzMj4AhA7rCAGe4KMBDvJI+AwstrUpVNzEA03Pprs=
 github.com/golang/protobuf v1.4.0-rc.4.0.20200313231945-b860323f09d0/go.mod h1:WU3c8KckQ9AFe+yFwt9sWVRKCVIyN9cPHBJSNnbL67w=
 github.com/golang/protobuf v1.4.0/go.mod h1:jodUvKwWbYaEsadDk5Fwe5c77LiNKVO9IDvqG2KuDX0=
+github.com/golang/protobuf v1.4.2 h1:+Z5KGCizgyZCbGh1KZqA0fcLLkwbsjIzS4aV2v7wJX0=
```

Comment:
By the way, any reason for changes in the go.sum? Could you make sure that you have rebased from master and that these changes are required?

Reply:
Maybe missing in an earlier commit?

```
> git co account_protected_export
Switched to branch 'account_protected_export'
> git rebase origin/master
Current branch account_protected_export is up to date.
> go mod vendor
> go mod tidy
> git status
On branch account_protected_export
Untracked files:
  (use "git add <file>..." to include in what will be committed)
	mytest
	server.test
nothing added to commit but untracked files present (use "git add" to track)
>